### PR TITLE
MRO Control Zero F7 OEM - Update

### PR DESCRIPTION
**Pull request by ryanjAA**
MRO Control Zero F7 OEM - Serial Mapping Update

This updates the serial mapping to be inline with the other Control Zero boards and also enables the CAN2 silent pin.

### CODE REVIEW

**Comment by dagar** [comment]
This looks like changes on top of the ctrl-zero f7 oem board support you already added? Try merging current master into this branch to resolve the conflicts.

**Comment by ryanjAA** [comment]
Yes - my bad. I was working on a different fork. I just merged master into the branch.

File: boards/mro/ctrl-zero-f7-oem/default.cmake [hunk 1]
```diff
@@ -109,7 +109,10 @@ px4_add_board(
 		top
 		topic_listener
 		tune_control
+<<<<<<< HEAD
```
Comment (dagar) [inline boards/mro/ctrl-zero-f7-oem/default.cmake:112]:
```suggestion
```

File: boards/mro/ctrl-zero-f7-oem/default.cmake [hunk 2]
```diff
@@ -109,7 +109,10 @@ px4_add_board(
...
+=======
```
Comment (dagar) [inline boards/mro/ctrl-zero-f7-oem/default.cmake:113]:
```suggestion
```

File: boards/mro/ctrl-zero-f7-oem/default.cmake [hunk 3]
```diff
@@ -109,7 +109,10 @@ px4_add_board(
...
 		uorb
+>>>>>>> master
```
Comment (dagar) [inline boards/mro/ctrl-zero-f7-oem/default.cmake:115]:
```suggestion
```

**Pull request merged**